README.md

```diff
@@ -18,7 +18,7 @@ Our regular calls are {insert times}. You can find the event in the global DRF c
 |    Name  | GH Handle |  Discord Handle |  LinkedIn  | 
 |    ---   |     ---   |      ---        |     ---    |
 | Divya Mohan | [@ divya-mohan0209](https://github.com/divya-mohan0209) |     @ ar.iadne    | [@ divya-mohan0209](https://linkedin.com/in/divya-mohan0209) |
-| Elizabeth R Acosta  | [@ lizacosta](https://github.com/lizacosta) |     @ but_on_disc0rd    | [@ lizacostalinkedin](https://www.linkedin.com/in/lizacostalinkedin/) |
+| Liz Acosta  | [@ liz-acosta](https://github.com/lizacosta) |     @ but_on_disc0rd    | [@ lizacostalinkedin](https://www.linkedin.com/in/lizacostalinkedin/) |
```

Comment:
Just one more change! The GH handle needs to be updated in the URL as well so from this: https://github.com/lizacosta to this: https://github.com/liz-acosta

Once that's updated feel free to merge!